ts/features/fci/saga/index.ts

```diff
@@ -35,15 +51,49 @@ export function* watchFciSaga(bearerToken: SessionToken): SagaIterator {
     }
   );
 
-  yield* fork(watchFciStartingSaga);
+  // handle the request of getting QTSP metadata
+  yield* takeLatest(fciLoadQtspClauses.request, function* () {
+    yield* call(handleGetQtspMetadata, fciClient.getQtspClausesMetadata);
+  });
+
+  yield* takeLatest(
+    fciLoadQtspFilledDocument.request,
+    function* (action: ActionType<typeof fciLoadQtspFilledDocument.request>) {
+      yield* call(
+        handleCreateFilledDocument,
+        fciClient.postQtspFilledBody,
+        action
+      );
+    }
+  );
+
+  yield* fork(watchFciQtspClausesSaga);
 
-  yield* fork(watchFciAbortingSaga);
+  yield* fork(watchFciStartSaga);
+
+  yield* fork(watchFciAbortSaga);
```

Comment:
why using the `fork` having three  long running task instead of subscribing a `takeLatest`?